webapp/components/signup/components/signup_email.jsx

```diff
@@ -40,7 +40,7 @@ export default class SignupEmail extends React.Component {
         this.state = this.getInviteInfo();
     }
 
-    componentDidMount() {
+    static componentDidMount() {
```

Comment:
Turns out that it _doesn't_ work if the function is an override of a function in the parent class. I guess the linter isn't actually that smart 👎